src/kernels/kernelConnector.ts

```diff
@@ -114,7 +115,8 @@ export class KernelConnector {
         actionSource: KernelActionSource
     ) {
         const memento = serviceContainer.get<Memento>(IMemento, GLOBAL_MEMENTO);
-        const errorHandler = serviceContainer.get<IDataScienceErrorHandler>(IDataScienceErrorHandler);
+        // Error handler may not be available in web situation
+        const errorHandler = serviceContainer.tryGet<IDataScienceErrorHandler>(IDataScienceErrorHandler);
```

Comment:
Wait, I thought you mentioned above that it is defined, hence I don't think `tryGet` is necessary here

Reply:
Not 100% sure. This is safer for now.

Reply:
Is that the right link & comment. Still don't understand why its safer when its not necessary.

Reply:
aah its here https://github.com/microsoft/vscode-jupyter/issues/9784

Reply:
Yeah the hard part is I have to cause a kernel error in remote. I guess I could create a bad kernelspec